### ISSUE TITLE
get rid of "other" family

### ISSUE THREAD

**Issue by maelle** [igraph/rigraph#700]
or at least change its title so it isn't "Other other" :grin:

**Comment by ntamas** [comment]
Where's this in the documentation? I can't find this on the site now.

**Comment by maelle** [comment]
https://maelle.github.io/rigraph/reference/index.html#other

from the script https://github.com/igraph/rigraph/blob/main/R/other.R

**Comment by ntamas** [comment]
It's now called "Miscellaneous" in the "Utilities" section on https://r.igraph.org/reference/index.html#miscellaneous - is that not up-to-date?

**Comment by maelle** [comment]
Yes but in the local docs and pkgdown individual reference pages you get https://r.igraph.org/reference/convex_hull.html#see-also

I wonder whether they should be in distinct families/no family at all.

**Comment by ntamas** [comment]
Okay, so how are these labels constructed? I can see in other places (e.g., [here](https://r.igraph.org/reference/constraint.html) ) that it seems to use the name of the file in which the function being referenced is to be found. Is there a way to customize this? For instance, I'd be perfectly fine with having a "See also" section that does not refer to the filename at all but simply lists the functions that we want to refer to.

**Comment by maelle** [comment]
We used the filename as a proxy when adding these labels automatically for the first draft of the reference configuration. so yes now they can be edited as one wants, as long as one modifies `_pkgdown.yml` in parallel (`pkgdown::check_pkgdown()` helps checking that file).*

* says the contributor who forgot to remove `srand()` from the pkgdown config file :see_no_evil:
